.github/workflows/zizmor.yml

```diff
@@ -0,0 +1,21 @@
+name: GitHub Actions Security Analysis with zizmor 🌈
+
+on:
+  pull_request:
+    branches: ["**"]
+
+jobs:
+  zizmor:
+    name: zizmor latest via PyPI
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+
+      - name: Install the latest version of uv
+        uses: astral-sh/setup-uv@v4
+
+      - name: Run zizmor 🌈
+        run: uvx zizmor ./.github/workflows
```

Comment:
Change to run against `.` for consistency with the automated version, perhaps.

Reply:
The automated runs on `.`? Ok will do. change

Reply:
Done in 554a49d3130493f4040f92be39f1d8f19aed0833